docs/Walkthrough/Working with the Catalog.md

```diff
@@ -22,7 +22,7 @@ You can also browse the underlying S3 files using the files tab.
 
 ### Search
 
-Catalogs also enable you to search the contents of your bucket. We support both unstructured  (e.g. "`San Francisco`") and structured (e.g. "`metadata_key: metadata_value`") search. Hits are previewed right in the search results.
+Catalogs also enable you to search the contents of your bucket. We support both unstructured  (e.g. "`San Francisco`") and structured with Query String Queries [ES 6.7](https://www.elastic.co/guide/en/elasticsearch/reference/6.7/query-dsl-query-string-query.html#query-string-syntax) (e.g. "`metadata_key: metadata_value`") search. Hits are previewed right in the search results.
```

Comment:
i think it makes more sense to make "Query String Queries" into a clickable link, not "ES 6.7", i.e. `[Query String Queries](https://www.elastic.co/guide/en/elasticsearch/reference/6.7/query-dsl-query-string-query.html#query-string-syntax)` (dont think it's necessary to refer to a specific version in the text at all tbh).
also, there's an extra space after `unstructured`, so maybe fix this while you're at it